### DISCUSSION TITLE
Spying of local chat

### DISCUSSION THREAD

**Discussion by mare-on** [Zrips/CMI#3444] [Feature Requests]
CMI has a lot of great features for spying the players.
In game there can be "local chat" and "global chat" where the key is "!".

Here comes the problem... CMI has permission "cmi.chat.rangebypass" which can be useful for admins to send message via "local chat" as a global messages. But there is nothing like "cmi.chat.rangespy" which could works like "commandspy" or "socialspy".

This absence is very bad by my opinion because admins aften travel between players and are not in GeneralRange so they can't see all messages from players. If there would be added this feature which is one of the basic functions for example in Essentials, that would be awesome.

**Comment by div2005** [comment]


**Reply by VitSun2001** [reply]
up

**Comment by VitSun2001** [comment]
Up

**Comment by div2005** [comment]
up
